climakitae/core/data_interface.py

```diff
@@ -946,122 +1037,148 @@ def _update_unit_options(self):
+            ) and ("Historical Reconstruction" in self.scenario_historical):
+                data_warning = """Historical Reconstruction data is not available with SSP data.
+                Try using the Historical Climate data instead."""
+
+            elif (
+                (  # Warn user if no data is selected
+                    all("SSP" not in one for one in self.scenario_ssp)
+                )
+                and (
+                    not True
+                    in ["Historical" in one for one in self.scenario_historical]
+                )
+                and (self.scenario_ssp != ["n/a"])
+                and (self.scenario_historical != ["n/a"])
+            ):
+                data_warning = "Please select as least one dataset."
 
-        elif (
-            (  # If both historical options are selected, warn user the data will be cut
+            elif (  # If both historical options are selected, warn user the data will be cut
                 "Historical Reconstruction" in self.scenario_historical
-            )
-            and ("Historical Climate" in self.scenario_historical)
-        ):
-            data_warning = """The timescale of Historical Reconstruction data will be cut 
-            to match that of the Historical Climate data if both are retrieved."""
+            ) and (
+                "Historical Climate" in self.scenario_historical
+            ):
+                data_warning = """The timescale of Historical Reconstruction data will be cut 
+                to match that of the Historical Climate data if both are retrieved."""
 
-        # Warnings based on time slice selections
-        if (not True in ["SSP" in one for one in self.scenario_ssp]) and (
-            "Historical Climate" in self.scenario_historical
-        ):
-            if (self.time_slice[0] < historical_climate_range[0]) or (
-                self.time_slice[1] > historical_climate_range[1]
+            # Warnings based on time slice selections
+            if (all("SSP" not in one for one in self.scenario_ssp)) and (
+                "Historical Climate" in self.scenario_historical
             ):
-                data_warning = bad_time_slice_warning
-        elif True in ["SSP" in one for one in self.scenario_ssp]:
-            if not True in ["Historical" in one for one in self.scenario_historical]:
-                if (self.time_slice[0] < self.ssp_range[0]) or (
-                    self.time_slice[1] > self.ssp_range[1]
+                if (self.time_slice[0] < historical_climate_range[0]) or (
+                    self.time_slice[1] > historical_climate_range[1]
                 ):
                     data_warning = bad_time_slice_warning
-            else:
-                if (self.time_slice[0] < historical_climate_range[0]) or (
-                    self.time_slice[1] > self.ssp_range[1]
+            elif True in ["SSP" in one for one in self.scenario_ssp]:
```

Comment:
same here

Reply:
leaving as-is for now but we can change in a future PR